PolyDock/PolyDock/src/pd/ecs/cmp/tabsHeader/TabsHeaderWidget.cpp

```diff
@@ -226,6 +250,20 @@ AlignedBox2i DefaultTabsHeaderWidget::getTextRectAtIdx(int idx) const
 		Vector2i{ right + left, bottom + top });
 }
 
+// ---------------------------------------------------------------------------------------------------------
+AlignedBox2i DefaultTabsHeaderWidget::getCloseButtonRectAtIdx(int idx) const
+{
+	const AlignedBox2i tabRect = getTabRectAtIdx(idx);
+	const int top = tabRect.min().y() + 8;
+	const int right = tabRect.max().x() - 8;
+	const int bottom = tabRect.max().y() - 8;
+	const int left = right - (bottom - top);
```

Comment:
this is cryptic :/ Either math is weird or I it is some shortcut, which can baffle a developer in future